app/controllers/rb_master_backlogs_controller.rb

```diff
@@ -40,8 +40,11 @@ class RbMasterBacklogsController < RbApplicationController
 
   def index
     @owner_backlogs = Backlog.owner_backlogs(@project)
-    @sprint_backlogs = Backlog.sprint_backlogs(@project)
-
+    unless params[:show_sub] and params[:show_sub].to_i == 1
+      @sprint_backlogs = Backlog.sprint_backlogs(@project)
+    else
+      @sprint_backlogs = Backlog.sprint_backlogs(@project,true)
+    end
```

Comment:
@ulferts 

First of all, thanks for the quickly reply and for spending the time for reviewing my rookie code.
As I am just a rails enthusiast, every coding advice is of great value for me.
Where do you recommend I put the 'with_subprojects?' helper? As far as I understand Rails 'helper' concept, only the views would be able to call it if I place it inside app/helpers, although I think controllers and views should be able to call this helper.